Motion_planning/README.md

```diff
@@ -0,0 +1,31 @@
+# Resources for motion planning
+
+## Courses:
+### Beginners:
+
+1. https://www.edx.org/course/autonomous-mobile-robots
+2. https://www.coursera.org/learn/robotics-motion-planning
+
+### Intermediate:
+
+1. https://www.coursera.org/learn/motion-planning-self-driving-cars
```

Comment:
Please make in this format `[Robot motion planning intro](http://ais.informatik.uni-freiburg.de/teaching/ss11/robotics/slides/18-robot-motion-planning.pdf)`

Reply:
All these links are self-explanatory that's why they are not hyperlinked.